app/src/components/trace/types.ts

```diff
@@ -4,7 +4,7 @@
 export interface ISpanItem {
   name: string;
   spanKind: string;
-  statusCode: SpanStatusCodeType;
+  propagatedStatusCode: SpanStatusCodeType;
```

Comment:
Same as before - this interface is a contract - not a definition of an API so should be un-touched.

Reply:
addressed in [c4f88f4](https://github.com/Arize-ai/phoenix/pull/1824/commits/c4f88f45f6b6b15167a7284e8484e93340e4b1ac)